test/integration/spec/myaccount/password.ts

```diff
@@ -1,3 +1,26 @@
+jest.mock('../../../../lib/features', () => {
+  const actual = jest.requireActual('../../../../lib/features');
+  return {
+    ...actual,
+    isDPoPSupported: () => true
+  };
+});
+
+jest.mock('../../../../lib/oidc/dpop', () => {
+  const actual = jest.requireActual('../../../../lib/oidc/dpop');
+  const keyPair = actual.generateKeyPair();
+
+  return {
+    ...actual,
+    generateKeyPair: async () => await keyPair,
+    findKeyPair: async () => await keyPair,
+    createDPoPKeyPair: async () => {
+      const kp = await keyPair;
+      return  { keyPair: kp, keyPairId: 'foo' };
+    }
+  };
+});
```

Comment:
If the DPoP mocks are standard and always the same, consider creating a shared mock utility. If you expect there to be a number tests that require different mock config then it might not be worthwhile.

Reply:
Agreed, however this isn't as simple as it seems. I have to mock/polyfill crypto libraries, IndexedDB as well as the DPoP mocks. These integration tests are a rather small suite so I felt like adding a IndexedDB mock to the overall environment was fine. The same isn't true for the unit test suites